### PR TITLE
Add citation file

### PR DESCRIPTION
**Pull request by FelixSchwock**
Resolves issue #94

### CODE REVIEW

**Review by John-Ragland — APPROVED** [review]

**Review by Molkree — CHANGES_REQUESTED** [review]
File extension should be `.cff`, other than that looks good.

File: CITATION.ccf [hunk 1]
```diff
@@ -0,0 +1,20 @@
+ccf-version: 1.2.0
```
Comment (Molkree) [inline CITATION.ccf]: and inside the file too, @FelixSchwock 
```suggestion
cff-version: 1.2.0
```

**Review by Molkree — CHANGES_REQUESTED** [review]

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,20 @@
+ccf-version: 1.2.0
```
Comment (Molkree) [inline CITATION.cff:1]: and inside the file too, @FelixSchwock 
```suggestion
cff-version: 1.2.0
```

**Review by Molkree — CHANGES_REQUESTED** [review]

File: CITATION.cff (hunk 2 above)
Comment (FelixSchwock) [reply CITATION.cff:1]: Ah thank you, I missed that.

**Review by Molkree — APPROVED** [review]

**Pull request merged**